src/core/server/server.ts

```diff
@@ -478,4 +461,125 @@ export class Server {
+                  _meta: {
+                    description:
+                      'Number of seconds the Node.js process has been running until the constructor finished',
+                  },
+                },
+              },
+            },
+            preboot: {
+              properties: {
+                start: {
+                  type: 'float',
+                  _meta: {
+                    description:
+                      'Number of seconds the Node.js process has been running until `preboot` was called',
+                  },
+                },
+                end: {
+                  type: 'float',
+                  _meta: {
+                    description:
+                      'Number of seconds the Node.js process has been running until `preboot` finished',
+                  },
+                },
+              },
+            },
+            setup: {
+              properties: {
+                start: {
+                  type: 'float',
+                  _meta: {
+                    description:
+                      'Number of seconds the Node.js process has been running until `setup` was called',
+                  },
+                },
+                end: {
+                  type: 'float',
+                  _meta: {
+                    description:
+                      'Number of seconds the Node.js process has been running until `setup` finished',
+                  },
+                },
+              },
+            },
+            start: {
+              properties: {
+                start: {
+                  type: 'float',
+                  _meta: {
+                    description:
+                      'Number of seconds the Node.js process has been running until `start` was called',
+                  },
+                },
+                end: {
+                  type: 'float',
+                  _meta: {
+                    description:
+                      'Number of seconds the Node.js process has been running until `start` finished',
+                  },
+                },
+              },
+            },
+          },
+          _meta: {
+            description:
+              'Number of seconds the Node.js process has been running until each phase of the server execution is called and finished.',
+          },
+        },
+      },
+    });
+  }
+
+  /**
+   * Reports the new and legacy KIBANA_STARTED_EVENT.
+   * @param analyticsStart The {@link AnalyticsServiceStart}.
+   * @private
+   */
+  private reportKibanaStartedEvents(analyticsStart: AnalyticsServiceStart) {
+    // Report the legacy KIBANA_STARTED_EVENT.
+    analyticsStart.reportEvent(KIBANA_STARTED_EVENT, { uptime_per_step: this.uptimePerStep });
+
+    const ups = this.uptimePerStep;
+
+    // Report the metric-shaped KIBANA_STARTED_EVENT.
+    reportMetricEvent(analyticsStart, {
+      eventName: KIBANA_STARTED_EVENT,
+      duration: ups.start!.end - ups.constructor!.start,
```

Comment:
We need to round all durations and convert to ms to match the field def.
`Math.round((ups.start!.end - ups.constructor!.start) * 1000);`